internal/ee/service/alert_evaluation_test.go

```
@@ -128,16 +103,11 @@ func (s *MeterUsageTrackingEvaluationSuite) TestCheckSpendBreachForEvent_NoAlert
 
-	event := &events.Event{
-		ID:                 "event_no_alert_settings",
-		ExternalCustomerID: s.customer.ExternalID,
-	}
-
 	// No alert_settings row exists for this subscription, so this must return before ever
 	// attempting a billing calculation — which would otherwise fail/panic on this bare fixture
 	// (no price/meter usage data behind it).
 	s.NotPanics(func() {
-		NewAlertService(s.svc.ServiceParams).EvaluateSpendBreachForEvent(ctx, event, s.customer)
+		NewAlertService(s.svc.ServiceParams).EvaluateSpendAlertsForCustomer(ctx, s.customer)
```

golangci-lint (2.12.2)

[error] Error return value of `(github.com/flexprice/flexprice/internal/ee/service.AlertService).EvaluateSpendAlertsForCustomer` is not checked

(errcheck)